ha/k8s_setup/ha_start.py

```diff
@@ -13,7 +15,8 @@
                                     )
 
 my_parser.add_argument('--services', '-s', help='service_name', required=True, action='store')
-my_parser.add_argument('--config', '-c', help='config file', required=False, action='store')
+my_parser.add_argument('--config', '-c', help='config file url', required=False, action='store')
+my_parser.add_argument('--debug', '-d', help='debug level {0|1}', type=int, nargs='?', const=1, default=0, required=False, action='store')
```

Comment:
So, just for my understanding: 
If script is executed with:
-d/--debug and no value: debug value will be 1
no debug argument at all: debug value will be 0
And if given, that value will be used!
So, why not to give 1 for both const and default? Any reason to give 1 and 0 respectively for const and default?

Reply:
if i give 1 for default then debug value will be always 1 even if you don't want it to be.
I mean if you don't provide option as '-d' or '--debug' still it will be set to 1.
please check attached test file or https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/956891431/Debug+option+for+cortx-ha you will understand why the default value should be 0.

Reply:
So, I think then the default will always be 0. Otherwise, if I execute with -d or --debug (without giving value), debug will be True which I may not want.

Reply:
E.g.
ha_hart -s <service_name> -c <config_url>
ha_hart -s <service_name> -c <config_url> -d 0
ha_hart -s <service_name> -c <config_url> --debug 0
all above same and here debug level will be zero by default and even if you provide it specifically.
ha_hart -s <service_name> -c <config_url> -d
ha_hart -s <service_name> -c <config_url> --debug
ha_hart -s <service_name> -c <config_url> -d 1
ha_hart -s <service_name> -c <config_url> --debug 1
In this all case debug level is set to 1

Reply:
I got this. But if someone used just -d mistakenly (debug or auto restart is not intended). Then also, debug will be 1. But, as Archana mentioned, this we can make it available separately as part of dev build. So, its not the problem then. 
Fine for now. Thanks!

Reply:
ok , I got your point my intetion putting 1 as const or default with -d option is no other level are in use so -d means will help to debug and caller may not to look into this level thing.
But your point is valid will keep a note and first will understand arachana's point as I need to do for that thing.